reference.md

```diff
@@ -1,6 +1,6 @@
 ---
-layout: reference
-permalink: /reference/
+layout: page
```

Comment:
you want to keep layout to `reference` here otherwise the page won't look quite right.

Reply:
done. Sorry for the confusion.